runtime/moonbase/src/lib.rs

```diff
@@ -306,7 +308,8 @@ impl pallet_balances::Config for Runtime {
 	type FreezeIdentifier = ();
 	type MaxFreezes = ConstU32<0>;
 	type RuntimeHoldReason = RuntimeHoldReason;
-	type MaxHolds = ConstU32<0>;
+	type RuntimeFreezeReason = RuntimeFreezeReason;
+	type MaxHolds = ConstU32<1>;
```

Comment:
Why `MaxHolds` needs to be 1 now?

Reply:
Because pallet preimage use Holds instead of reserve now